packages/dds/merge-tree/src/mergeTreeNodes.ts

```diff
@@ -45,24 +45,47 @@ import {
 import { SegmentGroupCollection } from "./segmentGroupCollection";
 import { PropertiesManager, PropertiesRollback } from "./segmentPropertiesManager";
 
+/**
+ * Common properties for a node in a merge tree.
+ */
 export interface IMergeNodeCommon {
     parent?: IMergeBlock;
     /**
      * The length of the contents of the node.
      */
     cachedLength: number;
+    /**
+     * The index of this node in its parent's list of children.
+     */
     index: number;
+    /**
+     * A string that can be used for comparing the location of this node to other `MergeNode`s in the same tree.
+     * `a.ordinal < b.ordinal` if and only if `a` comes before `b` in a pre-order traversal of the MergeTree.
+     */
     ordinal: string;
     isLeaf(): this is ISegment;
```

Comment:
Docs?

Reply:
I intentionally scoped this to fields defining state of merge tree nodes, can always improve later if we really think it's valuable for developer understanding. Not particularly high priority for me though.